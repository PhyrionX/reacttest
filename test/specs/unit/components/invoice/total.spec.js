import React from 'react';
import { render } from '@testing-library/react'
import Total from '../../../../../src/components/invoice/total';

describe('<Total />', () => {
  it('should mount empty component', () => {
    const { asFragment } = render(<Total />);
    
    expect(asFragment()).toMatchSnapshot();
  })

  it('should mount with total value', () => {
    const { asFragment } = render(<Total total={ 248 }/>);
    
    expect(asFragment()).toMatchSnapshot();
  })
})
