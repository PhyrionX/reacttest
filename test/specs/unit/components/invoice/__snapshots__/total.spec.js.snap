// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Total /> should mount empty component 1`] = `
<DocumentFragment>
  <div
    class="billin-total"
  >
    <span>
      Total
    </span>
    <span>
       €
    </span>
  </div>
</DocumentFragment>
`;

exports[`<Total /> should mount with total value 1`] = `
<DocumentFragment>
  <div
    class="billin-total"
  >
    <span>
      Total
    </span>
    <span>
      248 €
    </span>
  </div>
</DocumentFragment>
`;
